ui/src/app/views/homepage/component.html

```diff
@@ -1,80 +1,135 @@
+        </div>
+    </div>
+    <div *ngIf="firstTimeUser">
+        <div class="row">
+            <div class="col">
+                <div class="card double left">
+                    <div class="row">
+                        <img class="icon analyzeIcon" src="/assets/icons/Analyze.svg">
+                    </div>
+                    <div class="row analyzeHeading">Analyze and Publish with Workspaces
+                    </div>
+                    <div class="row">
+                        <label class="text">Dolor sit amet consectetuer adipiscing sed diam
+                            euismod tincidunt ut laoreet dolore.</label>
+                    </div>
+                    <div class="row heading">
+                        <button class="btn btn-homepage" (click)="addWorkspace()">CREATE WORKSPACE</button>
+                    </div>
                 </div>
-                <button class="profile-update-button" (click)="navigateToProfile()">UPDATE</button>
             </div>
-            <div class="pie-chart-box">
-                <ngx-charts-pie-chart
-                        [scheme]="colorScheme"
-                        [results]="spinnerValues"
-                        [doughnut]="true"
-                        [tooltipDisabled]="true">
-                </ngx-charts-pie-chart>
-                <div class="percent-text">
-                    {{completedTasksAsPercentage}}%
+            <div class="col">
+                <div class="card double right">
+                    <div class="row">
+                        <img class="icon learnIcon" src="/assets/icons/learn-icon.svg">
+                    </div>
+                    <div class="row heading learnHeading">
+                        Learn Researcher Workbench
+                    </div>
+                    <div class="row">
+                        <label class="text">Dolor sit amet consectetuer adipiscing.</label>
+                    </div>
+                    <div class="row heading">
+                        COMING SOON...
+                    </div>
                 </div>
             </div>
         </div>
     </div>
+    <div *ngIf="!firstTimeUser">
+        <div class="row">
+            <div class="card single">
+                <div class="row">
+                    <div class="col">
+                        <img src='/assets/icons/Analyze.svg' class="iconImage">
+                    </div>
+
+                    <div class="col analyzeSingleHeader">Analyze and Publish With Workspaces</div>
+                    <div class="col">
+                        <div class="row linkGroup">
+                            <div class="col">
+                                <a class="links" [routerLink]="" (click)="addWorkspace()">Create a
+                                    workspace</a>
+                            </div>
+                            <div class="col">
+                                <a class="links" [routerLink]="">See All Notebooks</a>
```

Comment:
Can we put a disabled on this or something? Or some sort of messaging. 

Also, I assume you talked with Lou and Fritz on this, but I am very hesitant to put in any code and pages where we don't have functionality or views yet.